features/az-cn-ir.geojson

```diff
@@ -0,0 +1,247 @@
+      [45.07151098606428, 41.32823894985094],
+      [45.245215836533134, 41.0134235932496],
+      [45.4366628066322, 41.01393848136652],
+      [45.478126452826274, 40.93355363656485],
+      [45.976637356833066, 40.21808399137643],
+      [46.504788063365424, 39.58379391501532],
+      [46.48607977421085, 39.483860662078996],
+      [46.532510885445504, 38.89551679924722],
+      [46.50574141335974, 38.88638987604563],
+      [46.49711451188654, 38.88987442933396],
+      [46.4644052142454, 38.89054744402799],
+      [46.45618933381934, 38.8990722499405],
+      [46.41455981610599, 38.892856023013735],
+      [46.38761386940439, 38.90594874584582],
+      [46.36897739926482, 38.90999802692869],
+      [46.365346918068894, 38.916212753191246],
+      [46.33957516915916, 38.917143940044525],
+      [46.33330780250233, 38.90901621246587],
+      [46.31330015779227, 38.901689759802196],
+      [46.29251462706475, 38.90356821633816],
+      [46.27373074168614, 38.897600528210006],
+      [46.265345039470134, 38.88581896185329],
+      [46.256765003496554, 38.8831242896253],
+      [46.25212188668232, 38.87681635187829],
+      [46.22611506769337, 38.86379077396816],
+      [46.215283366555354, 38.8611603212054],
+      [46.21156847865029, 38.85443815452618],
+      [46.20112365523772, 38.85203601932619],
+      [46.19177439945429, 38.851198629631],
+      [46.18728807490035, 38.84308744481902],
+      [46.18178057916356, 38.84061251402116],
+      [46.17296815593323, 38.84055011453751],
+      [46.16603838675533, 38.84292125645794],
+      [46.159829634023964, 38.84108050840658],
+      [46.1546416772658, 38.84307523895484],
+      [46.14274142072506, 38.84174077557839],
+      [46.13957472132404, 38.84428852543209],
+      [46.14194104615143, 38.844559557282054],
+      [45.813423180428146, 39.54568424584261],
+      [44.83593898542438, 39.73432800976505],
+      [44.80149528828244, 39.654774987976765],
+      [44.61243934304861, 39.78784156447],
+      [43.190918132668315, 39.203350413010355],
+      [47.082513990774885, 28.278365829252408],
+      [61.743916386659805, 24.216562135369784],
+      [61.66253669826381, 25.37801887657247],
+      [61.84848857899976, 26.095736314562316],
+      [63.32884583852959, 27.14194681911502],
+      [61.55139517354044, 28.97753645826529],
+      [61.82407969306786, 30.96619814437142],
+      [60.873905363836286, 32.160404271666465],
+      [61.006505589281915, 34.7087128151538],
+      [61.3971485399226, 35.414961279989754],
+      [61.173636588566126, 36.518919448927804],
+      [59.15423173408337, 37.60605170132074],
+      [58.54611921520129, 37.69756305911791],
+      [58.405232856419275, 37.62976366376465],
+      [56.109734380422594, 38.4137452248973],
+      [54.65218822340282, 37.44910617782409],
+      [54.62249826191993, 37.449327082986386],
+      [53.211798427691946, 36.90999719120106],
+      [48.885303542763126, 42.06064324046332],
+      [48.415437922722475, 41.63699828114585]
+    ],
+    [
```

Comment:
This polygon has two outer rings. It should be a multipolygon geometry instead. GL JS has no problem with this weirdly-formed polygon, but for instance GitHub’s GeoJSON previewer makes a mess of it.